store/go.mod

```diff
@@ -1,11 +1,12 @@
-module cosmossdk.io/store
+module cosmossdk.io/store/v2
 
 go 1.20
 
 require (
 	cosmossdk.io/errors v1.0.0
 	cosmossdk.io/log v1.1.0
 	cosmossdk.io/math v1.0.1
+	cosmossdk.io/store v1.0.0-alpha.1
 	github.com/cometbft/cometbft v0.38.0-rc3
```

Comment:
any chance we can delete v1 entirely to remove this dependency from the start? streaming might be a blocker for this though, not sure

Reply:
I'm not sure where this comes from actually. Yeah I'd like to remove it. Let me find where/why this dep was introduced.